src/Microsoft.Fx.Portability.MetadataReader/DependencyFinderEngineHelper.cs

```diff
@@ -69,6 +71,10 @@ public void ComputeData()
                 }
             }
 
+            // Get the assembly info of System.Object and set it as assembly info for primitives
+            var systemObjectMemberDependency = MemberDependency.FirstOrDefault(t => string.Equals(t.MemberDocId, "T:System.Object", StringComparison.Ordinal) && _assemblyFilter.IsFrameworkAssembly(t.DefinedInAssemblyIdentity));
```

Comment:
I feel like we should check in the case that it returned the default value (null) and throw an exception if we are unable to locate the assembly info for `T:System.Object` because this would be a bug somewhere in our code.